integrationtests/tests/fixtures/real.rs

```diff
@@ -59,14 +93,22 @@ impl RealLightningTest {
 }
 
 impl RealLightningTest {
-    fn channel_balance(rpc: &LightningRPC) -> Amount {
-        let funds: u64 = rpc
-            .listfunds()
-            .unwrap()
+    async fn channel_balance(rpc: &mut ClnRpc) -> Amount {
+        let listfunds_req = requests::ListfundsRequest { spent: Some(false) };
+        let listfunds_resp = if let Response::ListFunds(data) =
+            rpc.call(Request::ListFunds(listfunds_req)).await.unwrap()
+        {
+            data
+        } else {
+            // TODO: better way?
+            panic!("called ListFunds but got something else")
```

Comment:
same here `cln-rpc response didn't match request. expected ListfundsResponse got something else`

Reply:
also put this in one sentence. If you also agree on the use of `panic!` here, I mark this as resolved.